lib/db.js

```diff
@@ -791,7 +768,6 @@ Db.prototype.ensureIndex = deprecate(function(name, fieldOrSpec, options, callba
 
 Db.prototype.addChild = function(db) {
   if (this.s.parentDb) return this.s.parentDb.addChild(db);
-  this.s.children.push(db);
```

Comment:
I think when removing the internal `children` array, we should also just completely remove the concept of a `Db` having "children". I know it's a bit out of scope for the original ticket, but it seems like not much extra work.

Reply:
Done!